BeagleBone/AI/pru/init_pins_empty.h

```diff
@@ -0,0 +1,6 @@
+// Defins an empty table to write_init_pins wont' complain
+
+#pragma DATA_SECTION(init_pins, ".init_pins")
+#pragma RETAIN(init_pins)
+const char init_pins[] =
```

Comment:
I'm opposed to allocating variables in header files in general. We could probably make things smaller with a macro if it is the extra lines that you want to avoid.